runtime/doc/api.txt

```diff
@@ -2341,6 +2341,21 @@ nvim_ui_pum_set_height({height})                    *nvim_ui_pum_set_height()*
                 Parameters: ~
                     {height}  Popupmenu height, must be greater than zero.
 
+nvim_ui_pum_set({width}, {height}, {row}, {col})    *nvim_ui_pum_set_height()*
```

Comment:
maybe `set_size` would be better? Also, don't forget to adapt the toc entry (between the asterisks)!

Reply:
thanks! maybe `set_bounds` because it's both size and position?

Reply:
@clason this file is autogenerated, you don't need to worry about that.